src/content/themes/grid-portfolio.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Grid Portfolio"
-description: "A one page template for your Personal Portfolio Website built with Astro and TailwindCSS."
+title: "Bento-like Portfolio"
+description: "Astro-bento-portfolio is a minimal bento-like (almost) single page portfolio website made in Astro"
 image: "/src/content/themes/_images/grid-portfolio-hero.webp"
 author:
   url: "https://github.com/Ladvace"
   name: "Ladvace"
   avatar: "https://avatars.githubusercontent.com/u/31326058?v=4"
 categories:
   - "portfolio"
-repoUrl: "https://github.com/Ladvace/Grid-astro-portfolio"
+repoUrl: "https://github.com/Ladvace/astro-bento-portfolio"
 demoUrl: "https://gianmarco.xyz/"
 tools:
   - "solidjs"
-  - "tailwind"
+  - "unocss"
```

Comment:
We don't actually support `unocss` yet 👀 The tools list here maps back to a [list used for icons](https://github.com/withastro/astro.build/blob/bb84098acef2c4653f40dd6c31bb8bb1ccbd4ac6/src/content/config.ts#L30)

We'll need to add an option for unocss in a separate PR so it can be used here and shown in the main Theme Submission form

```suggestion
```

Reply:
sorry, my bad, I'm gonna open that PR ASAP

Reply:
I opened a PR: https://github.com/withastro/astro.build/pull/900